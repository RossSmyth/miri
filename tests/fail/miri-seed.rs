//@compile-flags: -Zmiri-seed=9223372036854775807
#![feature(core_intrinsics)]

fn main() {
    unsafe {
        core::intrinsics::breakpoint() //~ ERROR: trace/breakpoint trap
    };
}
