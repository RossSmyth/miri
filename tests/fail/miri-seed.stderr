error: abnormal termination: trace/breakpoint trap
  --> $DIR/miri-seed.rs:LL:CC
   |
LL |         core::intrinsics::breakpoint()
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ trace/breakpoint trap
   |
   = note: BACKTRACE:
   = note: inside `main` at $DIR/miri-seed.rs:LL:CC
   = note: This occured with miri seed: 9223372036854775807

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to 1 previous error

